### PR TITLE
Edited index.html

### PR DESCRIPTION
**Pull request by KalilK13**
Changed some text in the index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html> <head>
-<title>A whale of a greeting</title> </head>
+<title>A whale of a greeting!</title> </head>
 <body>
 <h1>hello, world</h1> <a href="about.html">About this project</a> <p>Call me Kalil.</p> <img src="images/breaching_whale.jpg">
 </body> </html>
```
Comment (amg15AD) [inline index.html:6]: this is Lowkey messy but cool

Comment (Nickwancode) [inline index.html:6]: Good job!

**Review by Nickwancode — APPROVED** [review]
Looks good

**Review by amg15AD — APPROVED** [review]
looks good no comments

**Pull request merged**